.travis.yml

```diff
@@ -33,12 +33,11 @@ before_script:
 script:
   - flake8 --ignore=E501 ansible_runner_service.py runner-service/
   - cd /tmp/tests && python -m unittest discover -p "test_*.py"
```

Comment:
Why we need to copy the tests to a tmp folder?... I think that we can run the unit tests and coverage in the same place.....

Reply:
we run a playbook - which hits the path too long issue for the ssh socket. Moving it to tmp works around this problem (it's fixed upstream but not in a release yet)